tests/unit/test_local_csp.py

```diff
@@ -32,7 +34,7 @@
 
 def test_meter_billing_ok(cba_config):
```

Comment:
docstrings?

Reply:
Seems unnecessary in tests?

Reply:
every other test has a docstring now I thought. If so let's be consistent

Reply:
I don't think so. I skimmed a couple other test modules and there's no function docstrings. Each one has a module docstring and there's some one line comments here and there. But not consistent docstrings. Will move forward on this for now. I don't see the need for docstrings in tests but if we want them they are needed everywhere it seems. So we could do that all at once?

Reply:
Some modules do. But I found an example:

```
def test_good_get_next_bill_time_monthly():
    """Test working utils.get_next_bill_time"""
    assert utils.get_next_bill_time(
...
```

The docstring seems unnecessary. It doesn't really add any value.

Reply:
fine but they are in test_adapter.py, test_config.py, and test_utlils.py so we should be consistent either way

Reply:
Either with or without is fine by me.